src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -13001,7 +13001,7 @@ private bool IsComplexElementInitializer()
 
         private bool IsNamedAssignment()
         {
-            return IsTrueIdentifier() && this.PeekToken(1).Kind == SyntaxKind.EqualsToken;
+            return IsTrueIdentifier() && this.PeekToken(1).Kind is SyntaxKind.EqualsToken or SyntaxKind.ColonToken;
```

Comment:
don't update this.  it is used in too many locations (like `using X = Y;`).    Instead add some new method like IsNamedMemberInitializer.  Then use that from IsInitializerMember.